l10n_th_account_tax_filing/data/ir.sequence.csv

```diff
@@ -0,0 +1,2 @@
+id,name,code,prefix,padding,implementation,use_date_range
```

Comment:
Import data with `xml` instead because file csv will import everytime when you upgrade this module.
This sequence should add i system 1 time and noupdate.

```

<?xml version="1.0" encoding="utf-8" ?>
<odoo noupdate="1">
    <record id="seq_account_tax_filing" model="ir.sequence">
        <field name="name">Account Tax Filing Sequence</field>
        <field name="code">account.tax.filing</field>
        <field name="padding" eval="5" />
        <field name="implementation">no_gap</field>
        <field name="prefix">ATF</field>
    </record>
</odoo>

```